validator/project_validator.go

```diff
@@ -466,6 +478,9 @@ func validateProjectConfigPlugins(pc *model.ProjectConfig) ValidationErrors {
 	if annotationSettings != nil {
 		webhook = &annotationSettings.FileTicketWebhook
 	}
+	if pc.BuildBaronSettings == nil {
+		return errs
```

Comment:
We should probably append an error to errs explaining what's wrong.

Reply:
`errs` at this stage will be empty so this is actually meant to be a no-op. Added a comment

Reply:
Can we return nil instead of errs if that's the case?

Reply:
Sure